integrations/terraform/gitbook-manifest.yaml

```diff
@@ -0,0 +1,27 @@
+name: terraform
+title: Terraform
+organization: gitbook
+visibility: private
+description: An integration to support syncing Terraform resources with GitBook.
+script: src/index.tsx
+scopes:
```

Comment:
These are just placeholder scopes for now; we'll need to review and add support for any missing ones.